x-pack/plugins/security_solution/public/management/components/policy_response/policy_response_wrapper.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { memo, useEffect, useState } from 'react';
+import { EuiEmptyPrompt, EuiLoadingContent, EuiSpacer, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { HostPolicyResponse } from '../../../../common/endpoint/types';
+import { PreferenceFormattedDateFromPrimitive } from '../../../common/components/formatted_date';
+import { useHttp } from '../../../common/lib/kibana';
+import { useGetEndpointPolicyResponse } from '../../hooks/endpoint/use_get_endpoint_policy_response';
+import { PolicyResponse } from './policy_response';
+import { getFailedOrWarningActionCountFromPolicyResponse } from '../../pages/endpoint_hosts/store/utils';
+
+export const PolicyResponseWrapper = memo<{
+  endpointId: string;
+}>(({ endpointId }) => {
+  const http = useHttp();
+  const { data, isLoading, isFetching, isError } = useGetEndpointPolicyResponse(http, endpointId);
+
+  const [responseConfig, setResponseConfig] =
+    useState<HostPolicyResponse['Endpoint']['policy']['applied']['response']['configurations']>();
+  const [responseActions, setResponseActions] =
```

Comment:
Just a note: the name `responseActions` may cause confusion with the Response Action feature